src/patternfly/layouts/Page/styles.scss

```diff
@@ -1,44 +1,127 @@
 // URL.com/guidelines#layout
 @import "../../patternfly-utilities";
 
-:root {
-  // hardcoded value of 15rem for now as we are working through use cases for this layout
-  --pf-l-page__sidebar-Width: 15rem;
-}
-
 .pf-l-page {
+  // Sidebar width does not relate to the width propierty
+  // it's hardcoded to the fix width of the sidebar
+  --pf-l-page__sidebar--md--Width: pf-size-prem(290px);
+  --pf-l-page__sidebar--sm--Width: 80%;
+  --pf-l-page__sidebar--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+  --pf-l-page__header-nav--sm--BackgroundColor: var(--pf-global--BackgroundColor--dark-transparent-100);
+  --pf-l-page__header-brand--md--BackgroundColor: var(--pf-global--BackgroundColor--dark-transparent-100);
+  --pf-l-page__main-section--BackgroundColor: var(--pf-global--BackgroundColor--light-300);
+  --pf-l-page__main-section--BoxShadow: var(--pf-global--BoxShadow--lg-right);
+  --pf-l-page__main-section--Padding: var(--pf-global--spacer--lg);
+  --pf-l-page__main-section--m-light--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+  --pf-l-page__main-section--m-dark--BackgroundColor: var(--pf-global--BackgroundColor--dark-transparent-200);
+  // --pf-l-page__main-section--m-dark--Padding: var(--pf-global--spacer--xl);
+
   display: grid;
+  min-height: 100vh;
   padding: 0;
   margin: 0;
-  grid-template-columns: var(--pf-l-page__sidebar-Width) 1fr;
+  grid-template-columns: var(--pf-l-page__sidebar--md--Width) 1fr;
   grid-template-rows: auto 1fr;
-  grid-template-areas: "header header" "main main";
+  grid-template-areas:
+    "header header"
+    "main main";
+
+  @media (min-width: $pf-global--breakpoint--md) {
+    &.pf-m-expanded {
+      grid-template-areas:
+        "header header"
+        "sidebar main";
+    }
+  }
 
-  // show sidebar above small breakpoint
-  @media (min-width: $pf-global--breakpoint--sm) {
-    grid-template-areas: "header header" "sidebar main";
+  &__sidebar {
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    width: var(--pf-l-page__sidebar--sm--Width);
+    grid-area: sidebar;
```

Comment:
The way this is positioned seems really funky to me, but I also can't figure out what the intended purpose of this thing is, and what the expected behavior is.